README.md

```diff
@@ -30,97 +41,37 @@ Features:
  * Bare-metal or OS
  * Porting layer provided
 
-The stack includes a comprehensive set of unit-tests.
-
-Prerequisites for all platforms
-===============================
-
- * CMake 3.13 or later
-
-Out-of-tree builds are recommended. Create a build directory and run
-the following commands from that directory. In the following
-instructions, the root folder for the repo is assumed to be an
-absolute or relative path in an environment variable named *repo*.
-
-The cmake executable is assumed to be in your path. After running
-cmake you can run ccmake or cmake-gui to change settings.
-
-
-Linux
-=====
+Limitations:
 
- * GCC 4.6 or later
+* IPv4 only
+* Only a single Ethernet interface
+* Only one stack instance
 
-```console
-user@host:~/build$ cmake $repo
-user@host:~/build$ make all check
-```
+This software is dual-licensed, with GPL version 3 and a commercial license.
+See LICENSE.md for more details.
 
-The clang static analyzer can also be used if installed. From a clean
-build directory, run:
 
-```console
-user@host:~/build$ scan-build cmake $repo
-user@host:~/build$ scan-build make
-```
+Requirements
+------------
+The the platform must be able to send and receive raw Ethernet Layer 2 frames,
+and the Ethernet driver must be able to handle frames of size ?? Bytes. It
+should also avoid copying data, for performance reasons.
 
-rt-kernel
-=========
+* cmake 3.13 or later
 
- * Workbench 2017.1 or later
+For Linux:
 
-Set the following environment variables. You should use a bash shell,
-such as for instance the Command Line in your Toolbox installation.
+* gcc 4.6 or later
+* Take precautions to improve timings
```

Comment:
This is a little vague (take precautions), can we refer to the chapter on linux timing?

Reply:
Done